static/js/components/CourseList.js

```diff
@@ -1,22 +1,30 @@
 import React from 'react';
 import _ from 'lodash';
-
+import Button from 'react-bootstrap/lib/Button';
 import {
   makeCourseStatusDisplay,
   makeCourseProgressDisplay,
 } from '../util/util';
+import STATUS_PASSED from '../constants';
```

Comment:
I think you will need to do a qualified import here, like this:

```js
import { STATUS_PASSED } from '../constants;'
```

since that module doesn't have `STATUS_PASSED` as a default export